python/rmm/rmm/python/logger.pyx

```diff
@@ -14,58 +14,9 @@
 
 import warnings
 
-from libcpp cimport bool
-
-
-cdef extern from "spdlog/common.h" namespace "spdlog::level" nogil:
-    cpdef enum logging_level "spdlog::level::level_enum":
-        """
-        The debug logging level for RMM.
-
-        Debug logging prints messages to a log file. See
-        `Debug Logging <https://github.com/rapidsai/rmm#debug-logging>`_
-        for more information.
-
-        Valid levels, in decreasing order of verbosity, are TRACE, DEBUG,
-        INFO, WARN, ERR, CRITICAL, and OFF. Default is INFO.
-
-        Examples
-        --------
-        >>> import rmm
-        >>> rmm.logging_level.DEBUG
-        <logging_level.DEBUG: 1>
-        >>> rmm.logging_level.DEBUG.value
-        1
-        >>> rmm.logging_level.DEBUG.name
-        'DEBUG'
-
-        See Also
-        --------
-        set_logging_level : Set the debug logging level
-        get_logging_level : Get the current debug logging level
-        """
-        TRACE "spdlog::level::trace"
-        DEBUG "spdlog::level::debug"
-        INFO "spdlog::level::info"
-        WARN "spdlog::level::warn"
-        ERR "spdlog::level::err"
-        CRITICAL "spdlog::level::critical"
-        OFF "spdlog::level::off"
-
-
-cdef extern from "spdlog/spdlog.h" namespace "spdlog" nogil:
-    cdef cppclass spdlog_logger "spdlog::logger":
-        spdlog_logger() except +
-        void set_level(logging_level level)
-        logging_level level()
-        void flush() except +
-        void flush_on(logging_level level)
-        logging_level flush_level()
-        bool should_log(logging_level msg_level)
-
-
-cdef extern from "rmm/logger.hpp" namespace "rmm" nogil:
-    cdef spdlog_logger& logger() except +
+from rmm.cpp._logger cimport logger
+
+from rmm.cpp._logger import logging_level
```

Comment:
I tried that but enum isn't accessible from python (can't import it). I also tried creating a python version of enum like, 
```
cpdef enum LoggingLevel:
    ...
```
But I get this error `UserWarning: enum class logging_level not importable from rmm.librmm.logger. You are probably using a cpdef enum declared in a .pxd file that does not have a .py or .pyx file.`

I think we can fix it by moving enum declaration from `librmm` to `pylibrmm`. But that's not ideal because there would `cdef extern from "spdlog/common.h" namespace "spdlog::level" nogil:` in `pylibrmm` (which we're trying to avoid). So should we we leave it as is or move the enum declaration to `pylibrmm` with the con I mentioned?

Reply:
Let's leave it where it is.